unidist/core/backends/mpi/core/controller/common.py

```diff
@@ -107,7 +108,7 @@ def release_rank(self, rank):
         )
 
 
-def pull_data(comm, owner_rank):
+def pull_data(comm, owner_rank=None):
```

Comment:
```suggestion
def pull_data(comm, owner_rank):
```
We can pass in MPI.ANY_SOURCE. The docstring should be elaborated with respect to this value.